### PR TITLE
Refactor design and display

### LINKED ISSUE #73: User should not see scroll bars around result image

### LINKED ISSUE #74: User should see all other suggestions from all selected activities
On the result page, the user only sees the filtered activities that match one activity, even if more than one activity is chosen.

### LINKED ISSUE #75: Remove unused footer

### LINKED ISSUE #76: User should see all content for suggestions on result page
User should not need a scroll bar or see overflowing text that describes each suggested activity.

### LINKED ISSUE #77: Update README installation instructions

### LINKED ISSUE #78: User should be clear on details shown via result page
As of now, the number of players and average play time are not always displayed correctly or consistently.  Change this so that the range of players is shown and the time is presented in minutes.

Additionally, each release date on the result page shows up as a date object.  Change this so that the date is presented in a user-friendly way.

### PR DESCRIPTION
**Pull request by nicolegooden**
#### What's this PR do?  

+ Corrects some misspelled words
+ Makes phrases and descriptors for activity details more clear
+ Removes scroll bars from result image
+ Updates README with proper installation instructions
+ Adjusts styling, specifically for the result component
+ Changes date object for release date of activities on result page to human-readable dates
+ Fixes bug related to the `similar picks` from the result page
  + only the last selected activity's suggestions were showing up
  + now, all suggestions from all activities exist as similar picks

#### Where should the reviewer start? 

Most of the changes occur in `ResultPage.js` and `ResultPage.scss`.

#### How should this be manually tested?  

Run this locally to ensure all functionality is working as expected, mostly for the form and result views.

#### Any background context you want to provide?  

I believe these small changes make for a better user experience, especially when viewing the suggested activity and similar picks via the `ResultPage` component.  Please let me know if there is anything to improve upon before merging!  Thanks.

#### What are the relevant tickets?  

Closes #73 
Closes #74 
Closes #75 
Closes #76 
Closes #77 
Closes #78

### CODE REVIEW

**Review by BlakeDonn — APPROVED** [review]
Looks good, thanks for touching things up :)

File: src/Components/ResultPage/ResultPage.js [hunk 1]
```diff
@@ -20,30 +21,25 @@ export const ResultPage = (props) => {
             || activity.album_title 
             || activity.title 
             || activity.name
-            || "no information"}
+            || "unknown"}
```
Comment (Atos20) [inline src/Components/ResultPage/ResultPage.js:24]: Change approved! 👍

File: src/Components/ResultPage/ResultPage.js [hunk 2]
```diff
@@ -20,30 +21,25 @@ export const ResultPage = (props) => {
...
           </h3>
+          {activity.release_date && 
+          <h3 className="title">{moment(activity.release_date).format('LL')}</h3>}
+          <h3 className="title">{activity.genre|| activity.type}</h3>
+          {activity.average_time &&
+           <h3 className="title">{`Play Time: ${activity.average_time} mins`}</h3>}
+          {activity.min_players  &&
+           <h3 className="title">
+             {`Players: ${activity.min_players} ${activity.max_players !== activity.min_players ? '-' + activity.max_players : ''}`}
+           </h3>}
```
Comment (Atos20) [inline src/Components/ResultPage/ResultPage.js:42]: this looks way cleaner thanks!!

File: src/Components/ResultPage/ResultPage.scss [hunk 3]
```diff
@@ -38,17 +38,16 @@
     .slider {
         grid-area: 2/ 2 / 5 / 5;
         color: white;
-        width: 300px;
+        width: 500px;
```
Comment (Atos20) [inline src/Components/ResultPage/ResultPage.scss:41]: I like how much more space you are adding here

**Review by Atos20 — APPROVED** [review]

**Pull request merged**